CashSDK/src/test/java/cash/just/sdk/AtmApiLoginTest.kt

```diff
@@ -38,30 +41,33 @@ class AtmApiLoginTest() : AbstractAtmApiTest() {
         val server = Cash.BtcNetwork.TEST_LOCAL
         var sessionKeyCreated: String = "";
         var error: String = "";
-
+        val countDownResponse : CountDownLatch = CountDownLatch(1)
         CashSDK.createSession(server, object : Cash.SessionCallback {
             override fun onSessionCreated(sessionKey: String) {
                 sessionKeyCreated = sessionKey;
+                countDownResponse.countDown();
```

Comment:
You might like to create a function for `CashSDK.createSession` to do not repeat this block in each test.
You might use @Before and start the tests always with a valid session.

Reply:
function is already created in class AbstractAtmApiTest.

AtmApiLoginTest.kt  is to test login this is way many login call can done with different case